hbt/selection/lepton.py

```diff
@@ -240,6 +240,12 @@ def tau_selection(
 
     TODO: there is no decay mode selection yet, but this should be revisited!
     """
+    # return empty mask if no tagged taus exists in the chunk
+    if not ak.any(events.Tau.pt):
+        # Convinint definition of empty mask with no substructure of the array (dtype=bool)
+        empty_mask = events.Tau.idDeepTau2017v2p1VSjet + events.Tau.pt > 0
+        return empty_mask, empty_mask
```

Comment:
```suggestion
        empty_mask = events.Tau.idDeepTau2017v2p1VSjet + events.Tau.pt > 0
        print("WARNING: this file has no Tau entry, proceeding with the creation of an empty mask for Taus")
        return empty_mask, empty_mask
```
Add a warning to alert users on files without Taus and add comment